test_v01/test_null_array.cpp

```diff
@@ -0,0 +1,99 @@
+// Man Group Operations Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <numeric>
+
+#include "sparrow_v01/layout/null_array.hpp"
+
+#include "doctest/doctest.h"
+
+#include "../test/external_array_data_creation.hpp"
+
+namespace sparrow
+{
+    arrow_proxy make_proxy_for_null_array(std::size_t size)
+    {
+        ArrowSchema sc;
```

Comment:
```suggestion
        ArrowSchema sc{};
```
To silence the static analysis warnings